installing/installing_ibm_cloud_public/user-managed-encryption-ibm-cloud.adoc

```diff
@@ -27,7 +27,7 @@ For more information about the `encryptionKey` parameter, see xref:../../install
 
 [NOTE]
 ====
-Be sure you have integrated Key Protect or Hyper Protect Crypto Services with your IBM Cloud Block Storage service. For more information, see the Key Protect link:https://cloud.ibm.com/docs/key-protect?topic=key-protect-integrate-services#grant-access[documentation] or Hyper Protect Crypto Services link:https://cloud.ibm.com/docs/hs-crypto?topic=hs-crypto-integrate-services[documentation].
+Be sure you have integrated Key Protect with your IBM Cloud Block Storage service. For more information, see the Key Protect link:https://cloud.ibm.com/docs/key-protect?topic=key-protect-integrate-services#grant-access[documentation].
```

Comment:
I would suggest changing to "Make sure you integrated..."

"Be sure" is not an action while "make sure" is. As for the tense, the IBM Style Guide at least implies a preference for the past simple (when past is necessary at all) at https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs